apps/web/ee/components/stripe/PaymentPage.tsx

```diff
@@ -49,7 +49,7 @@ const PaymentPage: FC<PaymentPageProps> = (props) => {
         embedIframeWidth = e.detail.data.iframeWidth as number;
       });
     }
-  }, [isEmbed]);
+  }, [date, isEmbed]);
```

Comment:
This will cause an infinite loop: Fixing similarly to this 1c4d2fae563c1a2e4c857b9f6c8ce8459621426a

Reply:
I missed converting this PR to Draft to avoid merging, as it was not ready yet.